src/ProjectTemplates/test/Templates.Tests/ApiTemplateTest.cs

```diff
@@ -35,7 +36,8 @@ public async Task ApiTemplateCSharp()
         await ApiTemplateCore(languageOverride: null);
     }
 
-    [ConditionalFact(Skip = "Unskip when Helix supports native AOT. https://github.com/dotnet/aspnetcore/pull/47247/")]
+    [ConditionalFact]
+    [SkipOnHelix("https://github.com/dotnet/aspnetcore/issues/47478", Queues = "OSX.13.Amd64.Open;Ubuntu.2004.Amd64.Open;Windows.11.Amd64.Client.Open;")]
```

Comment:
This set of queues seems duplicated a bunch of times. Maybe have a const in this project and append const here and elsewhere? It makes updating it easier in the future.